### PR TITLE
Use redux

### PR DESCRIPTION
**Pull request by paulinagonzalezc**
# Bookstore: Use Redux in React components

In this step I used Redux store to display books. I also added some UI elements to my components and made them dispatch actions. NOTE: editing an existing book and updating the progress is not part of this step. Neither is the styling of the application's components.

General requirements

* Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config)
* Make sure that you used correct flow [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md)
* Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

Project Requirements

* Make sure that list of books displays books from Redux store
* In your React component responsible for adding new books, add functionality for "Add new" button click
* In your React component responsible for removing books, add a functionality for "Remove" button click
* In your React component responsible for categories, add a functionality for "Check status" button click

### CODE REVIEW

File: src/components/Categories.js [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
+  return (
+    <div className="category-container">
+      <button
+        className="primary-button"
+        type="button"
+        onClick={() => dispatch(checkStatus())}
+      >
+        Check status
+      </button>
+    </div>
```
Comment (omarsalem7) [inline src/components/Categories.js]: - [x] Kindly consider when clicking on the check status button the message with `Under Construction` which you get from the state by using `useSelector`.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @paulinagonzalezc 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
-  Good descriptive README ✔️
- The Pull Request has a proper title and description. ✔️
- Correct git-flow ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Categories.js (hunk 1 above)
Comment (paulinagonzalezc) [reply src/components/Categories.js]: Hi @omarsalem7 , thank you for your comment, I will add the message.

**Review by ichala — APPROVED** [review]
Hi @paulinagonzalezc   ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**